### DISCUSSION TITLE
Table of existing short URLs on admin page is taking a long time to load

### DISCUSSION THREAD

**Discussion by ghost** [YOURLS/YOURLS#3270] [Q&A]
When I load the admin page, the page itself will load fine but the table that shows all of the shortened URLs that already exists will take a good 5 to 10 minutes to load, if it loads at all. This is happening in both Firefox and Chrome and I can load the database tables just fine in phpMyAdmin.

This just started, I haven't changed anything recently on my install.

**Comment by ozh** [comment]
See wiki page about troubleshooting, also enabling debug output, this might give some clues
